back/internal/usersvc/internal/userdb/qryuser.go

```diff
@@ -32,13 +33,14 @@ func (s *UserDB) upsertUser(ctx cx, sid string, x *pb.AddUserReq, y *pb.UserResp
 		return fmt.Errorf("invalid uid")
 	}
 
-	// todo: be able to link roleIDs to users.
-	stmt, err := s.db.Prepare("INSERT INTO user (user_id, username, email, email_hold, altmail, altmail_hold, full_name, avatar, password) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?) ON DUPLICATE KEY UPDATE user_id = ?, username = ?, email = ?, email_hold = ?, altmail = ?, altmail_hold = ?, full_name = ?, avatar = ?, password = ?")
-	if err != nil {
-		return err
-	}
-
-	_, err = stmt.Exec(
+	qry := `
+		INSERT INTO user (user_id, username, email, email_hold, altmail, altmail_hold,
+			full_name, avatar, password) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?) 
+		ON DUPLICATE KEY UPDATE user_id = ?, username = ?, email = ?, email_hold = ?,
+			altmail = ?, altmail_hold = ?, full_name = ?, avatar = ?, password = ?
```

Comment:
```suggestion
		INSERT INTO user (
			user_id, username, email, email_hold, altmail, altmail_hold, full_name, avatar, password
		)
		VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?) 
		ON DUPLICATE KEY UPDATE
			user_id = ?, username = ?, email = ?, email_hold = ?, altmail = ?, altmail_hold = ?,
			full_name = ?, avatar = ?, password = ?
```

My thought here is that breaking in the middle of a section is awkward. I don't suggest just applying my change. This is just one way to format the SQL - I could even imagine the column names each on their own line (though I wouldn't do that myself). I think the main thing here, for me, is the VALUES section being on it's own line.